### PR TITLE
Admin categories

### PR DESCRIPTION
**Pull request by lumosnysm**

### CODE REVIEW

**Comment by lumosnysm** [comment]
ready

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -10,7 +18,7 @@ def create
       else
         flash[:danger] = t ".message_danger"
       end
-      redirect_to categories_url
+      redirect_to request.referrer
```
Comment (mdmahmudurrahman) [inline app/controllers/admin/categories_controller.rb]: Can you try using redirect_back fallback_location method
[https://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_back](https://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_back)

File: app/controllers/admin/categories_controller.rb [hunk 2]
```diff
@@ -19,7 +27,7 @@ def destroy
       else
         flash[:danger] = t ".not_delete"
       end
-      redirect_to categories_url
+      redirect_to request.referrer
```
Comment (mdmahmudurrahman) [inline app/controllers/admin/categories_controller.rb]: The same here

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by lumosnysm** [comment]
ready

**Comment by mdmahmudurrahman** [comment]
commented
This pr can be (merged)
CC: [To:1818111] Nguyen Thi Huong (94)

**Pull request merged**